update_ui_example.cpp

```diff
@@ -75,10 +80,24 @@ void update_authorize(int32_t request)
 
 void update_progress(uint32_t progress, uint32_t total)
 {
-    uint8_t percent = (uint8_t)((uint64_t)progress * 100 / total);
+    uint8_t percent = progress * 100 / total;
+
+#ifdef MBED_APPLICATION_SHIELD
+    /* display progress */
```

Comment:
We shouldn't use app specific code into the Simple MCC lib.
Please remove references to MBED_APPLICATION_SHIELD

Reply:
Maybe use the trace options to display info on the serial port?